src/iapetos/collector/ring.clj

```diff
@@ -81,9 +81,12 @@
 
 ;; ### Latency/Count
 
+(defn- exception? [response] (instance? Exception response))
+
 (defn- ensure-response-map
-  [response]
+  [response options]
   (cond (nil? response)          {:status 404}
+        (exception? response)    {:status (:exception-status options)}
```

Comment:
I'd rather you pass `exception-status` as a param instead of the whole `options` map

Reply:
solved by 8465e56